ledger/sandbox/src/main/resources/db/migration/h2database/V8__Party_allocation_entries.sql

```diff
@@ -0,0 +1,38 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V8: party_allocation_entries
+--
+-- This schema version adds a table for tracking party allocation submissions
+---------------------------------------------------------------------------------------------------
+
+CREATE TABLE party_allocation_entries
+(
+    -- The ledger end at the time when the party allocation was added
+    ledger_offset    bigint              not null,
+    recorded_at      timestamp           not null, --with timezone
+    -- SubmissionId for the party allocation
+    submission_id    varchar primary key not null,
+    -- participant id that initiated the allocation request
+    participant_id   varchar             not null,
+    -- party
+    party            varchar,
+    -- displayName
+    display_name     varchar,
+    -- The type of entry, one of 'accept' or 'reject'
+    typ              varchar             not null,
```

Comment:
`entry_type`?

Reply:
`typ` is the naming that was used for ledger_entries and configuration_entries so stuck with that in case we want to collapse these to a single table in future, so they are as compatible in naming as possible

i agree entry_type is clearer but i think we should change all at the same time if we decide to, and we'll need to do a migration script for the other tables if we rename

Reply:
Agreed -- in that case shall we add a technical task for that into our backlog?

Reply:
> Agreed -- in that case shall we add a technical task for that into our backlog?

Yes let's do that